### PR TITLE
Improve video doc page

### PR DESCRIPTION
**Pull request by Wumpf**
### What

Mostly for updating it about the fact that we have ffmpeg now in the native viewer, but some more general polish and revamp.

This is the last ship-blocker for ffmpeg based h264 decoding, so this
* closes https://github.com/rerun-io/rerun/issues/7607

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/8007?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/8007?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!
* [x] If have noted any breaking changes to the log API in `CHANGELOG.md` and the migration guide

- [PR Build Summary](https://build.rerun.io/pr/8007)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

**Comment by Wumpf** [comment]
rendered
https://landing-27o0skl6o-rerun.vercel.app/docs/reference/video

File: docs/content/reference/video.md [hunk 1]
```diff
@@ -27,62 +27,71 @@ snippet: archetypes/video_auto_frames
...
+
+### Overview
+
+Codec support varies in the web & native viewer:
+
+|            | Browser | Native |
+| ---------- | ------- | ------ |
+| AV1        | ✅       | ✅      |
+| H.264/avc  | ✅       | ✅      |
+| H.265/hevc | 🚧       | ❌      |
```
Comment (Wumpf) [inline docs/content/reference/video.md:48]: maybe 🚧 doesn't quite cut it. It's not like we're working on it actively (we can't - hevc support is a legal issue for mozilla). But 🟧 is worse

File: docs/content/reference/video.md [hunk 2]
```diff
@@ -27,62 +27,71 @@ snippet: archetypes/video_auto_frames
...
+
+<!-- for web codecs see https://www.w3.org/TR/webcodecs-codec-registry/#video-codec-registry -->
+
+Details see below.
+
 When choosing a codec, we recommend [AV1](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Video_codecs#av1),
 as it seems to have the best overall playback support while also having very high compression quality.
-AV1 is also patent-free, and is the only codec we currently support in the native viewer (see [#7298](https://github.com/rerun-io/rerun/issues/7298)). H.264/avc is another popular choice, and native support for that is coming soon.
+
+Since AV1 can have very long encoding times, it is often not yet suitable for streaming.
```
Comment (zehiko) [inline docs/content/reference/video.md:59]: super ultra nit: often not yet -> often not

File: docs/content/reference/video.md (hunk 1 above)
Comment (Wumpf) [reply docs/content/reference/video.md:48]: trying 🔲 instead

**Pull request merged**